Framework/DataHandling/test/SaveGSSTest.h

```diff
@@ -159,6 +159,69 @@ class SaveGSSTest : public CxxTest::TestSuite {
     AnalysisDataService::Instance().remove(wsName);
   }
 
+  void test_2BankRalfAltFormat() {
+    // Tests saving a 2 bank instrument in RALF - Alternative format
+    const std::string wsName = "SaveGSS_RALF_ALT_ws";
+
+    // Choose a number where mod 4 != 0 to check it handles having
+    // not enough data for one line correctly
+    const int numBins = 10;
+    auto dataWs =
+        generateTestMatrixWorkspace(wsName, m_defaultNumHistograms, numBins);
+
+    auto outFileHandle = Poco::TemporaryFile();
+    const std::string outFilePath = outFileHandle.path();
+
+    auto alg = setupSaveGSSAlg(outFilePath, wsName, "RALF");
+    alg->setProperty("Data Format", "ALT");
```

Comment:
`SaveGSS(v1) property (Data Format) violates conventions` according to RHEL7, might be worth updating the wiki at https://www.mantidproject.org/Mantid_Standards#Properties to mention that spaces in the name are not allowed